### ISSUE TITLE
the sim crashes when I hit simulate

### ISSUE THREAD

**Issue by identitateliviu** [wowsims/cata#601]
**Describe the bug**

The sim (cata, 0.0.5) crashes when I hit simulate.

**To Reproduce**

1. Start the sim
2. Load saved character profile
3. Hit Simulate
4. Crash.

**Expected behavior**

I expected it to run.

**Sim Links and Screenshots**

**Desktop (please complete the following information):**
 - OS: windows11
 - Browser: Firefox
 - Version 126.0.1 (64-bit)
 - 
**Smartphone (please complete the following information):**

N/A

**Additional context**

Log attached.
[sim-log.txt](https://github.com/user-attachments/files/15527747/sim-log.txt)

**Comment by rosenrusinov** [comment]
@FelixPflaum any ideas on this one?

**Comment by FelixPflaum** [comment]
@rosenrusinov This looks like a hardware problem (corrupted stack memory) again. A similar report in discord recently turned out to be an unstable intel CPU.

The solution was: "Installed Intel XTU and lowered performance core ratio down from 56x to 55x 
sim working now"
Following directions from this link that someone posted: https://www.radgametools.com/oodleintel.htm

Maybe this is the same issue?

**Comment by identitateliviu** [comment]
Thank you.
I'll try this tonight.
Fyi, this does not affect the wotlk sim, which works fine - only the
cataclysm one.

**Comment by FelixPflaum** [comment]
Yeah, that's because the cata sim is multi threaded now and uses 100% of the CPU. That makes stability problems like these, or rather slightly unstable (automatic) overclocks, become a problem, while programs that do not load the CPU enough may run perfectly fine.

**Comment by identitateliviu** [comment]
Hello.

I'm only writing back to confirm that changing some BIOS settings fixed this issue (this is just for future reference, as I suspect other players may contact you with this) and to apologise for raising it in the first place; as embarrassing as it may be, I didn't realise that was what was causing the issue; I did look at the stack frame & top of the stack, but I didn't realise the issue was caused by the BIOS/CPU (I wasn't aware of the issue you linked from rad/oodle).

It is indeed a 13900k on an Asus motherboard. So much for buying "the latest and greatest", I guess.

Thank you, I'll try to close this.
